### PR TITLE
fix : Placeholders in delete note type dialog

### PR DESCRIPTION
**Pull request by pratyaksh1610**
## Purpose / Description
fix : Placeholders in delete note type dialog.

## Approach
Replace strings.

## How Has This Been Tested?

image
## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/notetype/ManageNotetypes.kt [hunk 1]
```diff
@@ -150,12 +150,12 @@ class ManageNotetypes : AnkiActivity() {
             AlertDialog.Builder(this@ManageNotetypes).show {
                 title(R.string.model_browser_delete)
                 message(messageResourceId)
-                positiveButton(R.string.dialog_ok) {
+                positiveButton(R.string.dialog_yes) {
```
Comment (david-allison) [inline AnkiDroid/src/main/java/com/ichi2/anki/notetype/ManageNotetypes.kt:153]: As per material guidelines, I believe this should be `delete/cancel` 

(I could only find this for V2)

> Don’t use action text that fails to indicate what the selection will do. “Cancel” and “Delete” better indicate what will occur in this dialog.

https://m2.material.io/components/dialogs

> The confirmation action is disabled until all mandatory fields are filled. Use descriptive verbs such as “Save,” “Send,” “Share,” “Update,” or “Create.” Don’t use vague terms such as “Done,” “OK” or “Close.”

https://m3.material.io/components/dialogs/guidelines

**Review by david-allison — CHANGES_REQUESTED** [review]

File: AnkiDroid/src/main/java/com/ichi2/anki/notetype/ManageNotetypes.kt (hunk 1 above)
Comment (pratyaksh1610) [reply AnkiDroid/src/main/java/com/ichi2/anki/notetype/ManageNotetypes.kt:153]: `Delete` would be a better option imo.

**Review by david-allison — APPROVED** [review]

**Review by lukstbit — APPROVED** [review]
Thanks, but before we can merge this you need to squash the 3 commits in this PR into a single one.

As a side note, using multiple commits it's only valid if each commit introduces a specific change(this doesn't apply here as the change is very small). Here's [one article](https://www.freecodecamp.org/news/git-squash-commits/) about squashing, there are  more if you search for "_git squashing_".

**Review by lukstbit — APPROVED** [review]
Thanks!

**Pull request merged**